main.go

```diff
@@ -105,6 +113,8 @@ func main() {
 		"The page size limiting the number of response for list operation to API Server")
 	fs.IntVar(&healthProbePort, flagHealthProbePort, defaultHealthProbePort,
 		"The port the health probes binds to.")
+	fs.StringVar(&tlsOpt.minVersion, "tls-min-version", "VersionTLS12", "Minimum TLS version supported. Value must match version names from https://golang.org/pkg/crypto/tls/#pkg-constants.")
```

Comment:
It would be best to leave the default behavior a no-op when tls-min-version is not specified to avoid any impact to existing deployments.

Reply:
If we set to "nil", it will fall back to default setting from k8sapiflag DefaultTLSVersion() https://github.com/kubernetes/component-base/blob/v0.26.2/cli/flag/ciphersuites_flag.go#L142 which is the same as tls1.2.